.github/workflows/main.yaml

```diff
@@ -0,0 +1,34 @@
+name: Main branch CI
+
+on:
+  pull_request:
+    branches:
+      - main
```

Comment:
How about removing the branch from the PR trigger? Is there any reason why we wouldn't want the CI to run for PRs to other branches, like release ones?
```suggestion
```